core/scripts.py

```diff
@@ -119,6 +124,10 @@ def __init__(self, _db=None, *args, **kwargs):
         if _db:
             self._session = _db
 
+        if services is None:
+            services = container_instance()
+        self._services = services
```

Comment:
Not sure I agree here, but I updated this one anyway.

Reply:
That's why it's "minor" and "style". I def won't be offended if you disagree or otherwise decide not to change it. I'll try to use "style pref" in the future, to make that more clear.

I generally like the ternary in these cases because it does two things:
1. Makes it more clear to me that the main thing happening is that `self._services` is gonna get something assigned, either a specified value or a default.
2. Makes it impossible for what's happening in ll. 127-128 and l. 129 above to get separated in the code.

(2) here is also why I'm a fan of the walrus operator, though I never suggest that anyone change that one, since it seems more controversial.